src/Epinova.ElasticSearch.Core.EPiServer/Extensions/ContentExtensions.cs

```diff
@@ -664,6 +663,15 @@ private static object GetIndexValue(IContentData content, PropertyInfo p, out bo
             }
         }
 
+        private static IEnumerable<IContent> GetFilteredItems(IContentData content, ContentArea contentArea)
```

Comment:
Change prefix verb: Get -> List?

Reply:
This could be a local function mabye, or would be ok if other methods calls this?

Reply:
Single usage. I'll adjust.